roles/gerrit/tasks/main.yml

```diff
@@ -0,0 +1,79 @@
+---
+- name: Install Java and python-bugzilla
+  package:
+    state: present
+    name: "{{ item }}"
+  with_items:
+  - java-1.8.0-openjdk
+  - python-bugzilla
+
+- name: Create a user for review
+  user:
+    name: "{{gerrit_user}}"
+    home: "{{gerrit_user_home}}"
+
+
+- name: Check if Gerrit is installed
+  stat:
+    path: "{{gerrit_user}}/{{gerrit_user_home}}"
+    register: gerrit_status
+
+- name: "Fail when Gerrit is not setup"
+  fail:
+    msg: "Please setup Gerrit at {{gerrit_user}}/{{gerrit_user_home}}"
+  when: gerrit_status.status.islnk is not defined
```

Comment:
This test will fail if "{{gerrit_user}}/{{gerrit_user_home}}" do not exist, gerrit_status would contains: 
```
{
        "changed": false, 
        "stat": {
            "exists": false
        }
}
```

So I think we need to just check if the file exists.